web/src/copy_and_paste.ts

```diff
@@ -454,6 +454,24 @@ export function paste_handler_converter(
 
             This is done because filter() function only processes HTMLElements
         */
+        let katex_node_count = 0;
+        for (const child of children) {
+            if (child.nodeType === 1) {
```

Comment:
I think you can create a const to represent the `1` value to avoid the magic numbers.

```suggestion
            const <typeName> = 1;
            if (child.nodeType === <typeName>) {
```

You can rename the `<typeName>` using the concrete type name that represents the `1` value.

Reply:
Don’t make a new constant for what already exists: `Node.ELEMENT_NODE`.